### DISCUSSION TITLE
Recreated existing stack in Dockge, but it won't go away.

### DISCUSSION THREAD

**Discussion by jlyfshhh** [louislam/dockge#274] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I had a docker-compose.yml set up for my media stack based off of trash's guides. This stack was called trash-guides. I copied those containers into a new stack via the dockge GUI and named the stack media. I brought down the old stack and ran a couple docker prune commands just to clean up. Then I hit save and started the new stack.

This resulted in the trash-guides stack coming back with the message that it isn't managed by dockge, and my new media stack says it's inactive. I can view the logs in the media stack tab and if I stop it, it actually brings down the old stack which makes it go away. Once I start the new media stack, it stays inactive but the old trash-guides stack comes back.

I have completely removed the old trash docker-compose, deleted all relevant networks, stopped the containers, checked the /opt/stacks folder and I can't really figure out where dockge is still detecting this old stack even though I created a new stack within dockge itself.

### 📝 Error Message(s) or Log

Old stack is active while new stack is inactive

### 🐻 Dockge Version

1.3.3

### 💻 Operating System and Arch

Synology DSM 7.3

### 🌐 Browser

Safari

### 🐋 Docker Version

Docker 20.10.23

**Comment by louislam** [comment, marked as answer]
Dockge uses this command to get the list of all stacks:

 `docker compose ls`

And make sure for your old stack, you have run `docker compose down` to completely remove it.

**Reply by jlyfshhh** [reply]
Thank you, I didn't know about that command and it is very helpful.

trash-guides        running(9)          /opt/stacks/media/compose.yaml

I can now see that even though it's using the new compose in /opt/stacks/media it still considers this to be the old stack for some reason. I tried bringing the stack down but it just comes back up as trash-guides. is there a cache or something I need to clear?

**Reply by jlyfshhh** [reply]
Fixed. I forced the stack name change to media with bringing it up using the -p flag. Thanks for your help!

**Reply by louislam** [reply]
Currently, Dockge requires the stack folder name to be the same as the project name.

In your case, `trash-guides` do not match the folder name `media`.
